src/log.js

```diff
@@ -0,0 +1,18 @@
+var winston = require('winston');
+require('winston-daily-rotate-file');
+
+var transport = new (winston.transports.DailyRotateFile)({
```

Comment:
Files to be created in a folder named "logs" so a folder location here should be provided. 
The logs folder should be outside source and be along with our package.json. Add logs folder in git ignore too.

Reply:
Please give relative path instead of absolute path from your system.